docs/architecture/translations.rst

```diff
@@ -0,0 +1,84 @@
+Model Translations
+==================
+
+.. note::
+
+    The Model Translations are in the early version. They are fully functional, however, translations can be created only via custom code.
+
+    An API and UI for the translation interface will be added in the future releases.
+
+Overview
+--------
+
+Model translations are available via *TranslationProxy* defined on the to-be-translated *Model*.
```

Comment:
It would be better for readability and for consistency to the rest of the docs, to use inline code instead of italics.